src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/DispatchBusinessProcessCallbackHandlerTest.java

```diff
@@ -32,23 +36,33 @@
 
         @Test
         void shouldDispatchBusinessProcess_whenStatusIsReady() {
+            BusinessProcess businessProcess = BusinessProcess.builder()
+                .camundaEvent("testCamundaEvent")
+                .activityId("testActivityId")
+                .processInstanceId("testProcessInstanceId")
+                .status(READY)
+                .build();
             CaseData caseData = CaseDataBuilder.builder()
-                .businessProcess(BusinessProcess.builder()
-                                     .camundaEvent("testCamundaEvent")
-                                     .activityId("testActivityId")
-                                     .processInstanceId("testProcessInstanceId")
-                                     .status(READY)
-                                     .build())
+                .businessProcess(businessProcess)
                 .build();
             CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
 
             var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
 
-            assertThat(response.getData()).extracting("businessProcess").extracting("status").isEqualTo(DISPATCHED);
-            assertThat(response.getData()).extracting("businessProcess").extracting("camundaEvent").isEqualTo(
-                "testCamundaEvent");
-            assertThat(response.getData()).extracting("businessProcess").extracting("activityId").isNull();
-            assertThat(response.getData()).extracting("businessProcess").extracting("processInstanceId").isNull();
+            assertThat(response.getData()).extracting("businessProcess")
+                .extracting("status").isEqualTo(DISPATCHED.name());
+            assertThat(response.getData())
+                .extracting("businessProcess")
+                .extracting("camundaEvent")
+                .isEqualTo("testCamundaEvent");
+            assertThat(response.getData())
+                .extracting("businessProcess")
+                .extracting("activityId")
+                .isNull();
+            assertThat(response.getData())
+                .extracting("businessProcess")
+                .extracting("processInstanceId")
+                .isNull();
```

Comment:
does .extracting not take multiple strings for assertions / can you not chain from .extracting business process

Reply:
I am afraid not. as we have condition on one property. we can not chain after that.

Reply:
done